package.json

```diff
@@ -50,27 +50,25 @@
     "@angular/forms": "5.0.2"
   },
   "devDependencies": {
+    "@angular/cli": "1.6.0",
     "@angular/compiler": "5.0.2",
     "@angular/compiler-cli": "5.0.2",
     "@angular/http": "5.0.2",
     "@angular/platform-browser": "5.0.2",
     "@angular/platform-browser-dynamic": "5.0.2",
     "@angular/router": "5.0.2",
-    "@ngtools/webpack": "1.8.2",
-    "@types/jasmine": "2.5.41",
+    "@types/jasmine": "2.5.51",
+    "@types/jasminewd2": "~2.0.2",
     "@types/node": "^6.0.46",
     "angular2-template-loader": "^0.6.0",
```

Comment:
I think it should be also safe to remove all webpack-demo-related packages, like:
```
angular2-template-loader
css-loader
ts-loader
etc...
```